src/Silk.Core/Utilities/HelpFormatter.cs

```diff
@@ -60,7 +60,7 @@ public override CommandHelpMessage Build()
             }
             else
             {
-                if (Command.IsExperimental()) embed.WithColor(DiscordColor.DarkRed).WithFooter("This command is in testing, and marked as Experimental! Please open a ticket if it breaks.");
+                if (Command.IsExperimental()) embed.WithColor(DiscordColor.DarkRed).WithFooter("\nThis command is in testing, and marked as Experimental! Please open a ticket if it breaks.");
```

Comment:
Why a newline in a footer?

Reply:
Looked a bit bunched with the footer on some Embeds; the footer and the text above looked close.